pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
What version of this are you shipping from the 1.0.0 branch? We should probably ensure we don't conflict with each other.